curriculum/Unit7/Lesson-702.md

```diff
@@ -275,14 +275,14 @@ Video
 - BJP 13-2, _Sorting_<br>
   <http://media.pearsoncmg.com/aw/aw_reges_bjp_2/videoPlayer.php?id=c13-2>
 
-- CSE 143, _Bubble Sort_ (optional) (8:34–13:38)<br>
-  <https://uw.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=1fdd364a-4d01-49cf-96b3-ce2f67b77dcf&start=514>
```

Comment:
Would it make sense to put these three CSE 143 videos in the reverse order, since that's the order they appear in in the original video? (That is: Selection, Insertion, Bubble)?